webapp/src/ts/providers/translate-utils.provider.ts

```diff
@@ -34,3 +40,23 @@ export class TranslateMessageFormatCompilerProvider extends TranslateCompiler {
     return translations;
   }
 }
+
+@Injectable()
+export class TranslateParserProvider extends TranslateDefaultParser {
```

Comment:
Maybe adding a comment of why we're doing this, like in the ticket.

Reply:
added